### ISSUE TITLE
[FEATURE] Support for minimum learning rate

### ISSUE THREAD

**Issue by tmostak** [h2oai/h2o-llmstudio#671]
### 🚀 Feature

Support for specification of a minimum learning rate

### Motivation

Often in the research literature minimum learning rates are set when fine-tuning a model using a cosine or linear schedule, preventing the learning rate from dropping to 0 by the end of training. It would be very helpful if this could be supported in the LLM Studio UX.

**Comment by pascal-pfeiffer** [comment]
There is actually an open issue for that at transformers: https://github.com/huggingface/transformers/issues/28441
Once merged there, we can quickly integrate in H2O LLM Studio, as we are already using the transformers schedulers.
